docs/csharp/quick-starts/branches-and-loops.yml

```diff
@@ -0,0 +1,241 @@
+
+    ```csharp
+    int a = 5;
+    int b = 3;
+    int c = 4;
+    if ((a + b + c > 10) && (a > b))
+    {
+        Console.WriteLine("The answer is greater than 10");
+        Console.WriteLine("And the first number is greater than the second");
+    }
+    else
+    {
+        Console.WriteLine("The answer is not greater than 10");
+        Console.WriteLine("Or the first number is not greater than the second");
+    }
+    ```
+    
+    The `&&` represents "and". It means both conditions must be true to execute
+    the statement in the true branch.  These examples also show that you can have multiple
+    statements in each conditional branch, provided you enclose them in `{` and `}`.
+    
+    You can also use  `||` to represent "or":
+
+    ```csharp
+    int a = 5;
+    int b = 3;
+    int c = 4;
+    if ((a + b + c > 10) || (a > b))
+    {
+        Console.WriteLine("The answer is greater than 10");
+        Console.WriteLine("Or the first number is greater than the second");
+    }
+    else
+    {
+        Console.WriteLine("The answer is not greater than 10");
+        Console.WriteLine("And the first number is not greater than the second");
+    }
+    ```
+
+- title: use loops to repeat operations
+  durationInMinutes: 4
+  content: |
+    Another important concept to create larger programs is **loops**. You'll
+    use loops to repeat statements that you want executed more than once. Try
+    this code in the interactive window:
+
+    ```csharp
+    int counter = 0;
+    while (counter < 10)
+    {
+      Console.WriteLine("Hello World! The counter is " + counter);
+      counter++;
+    }
+    ```
+
+    The `while` statement checks a condition and executes the statement
+    following the `while`. It will repeat checking the condition and
+    executing those statements until the condition is false.
+
+    There's one other new operator in this example. The `++` after
+    the `counter` variable is the **increment** operator. It adds 1
+    to the value of counter, and stores that value in the counter variable.
+
+    > [!IMPORTANT]
+    > Make sure that the `while` loop condition does switch to
+    > false as you execute the code. Otherwise, you create an
+    > **infinite loop** where your program never ends. Let's
+    > not demonstrate that, because the engine that runs your
+    > code will timeout and you'll see no output from your program.
+
+    The `while` loop tests the condition before executing the code
+    following the `while`. The `do` ... `while` loop executes the 
+    code first, and then checks the condition. It looks like this:
+
+    ```csharp
+    int counter = 0;
+    do
+    {
+      Console.WriteLine("Hello World! The counter is " + counter);
+      counter++;
+    } while (counter < 10);
+    ```
+    
+    This `do` loop and the earlier `while` loop work the same. 
+    
+    Let's move on to one last loop statement.
+
+- title: Work with the `for` loop.
```

Comment:
remove period and perhaps the inline code too?

![image](https://user-images.githubusercontent.com/12971179/30581701-e720838c-9cd5-11e7-86f9-a1dafafff781.png)